Backpack-SwiftUI/DynamicStack/Classes/BPKDynamicStack.swift

```diff
@@ -0,0 +1,42 @@
+import SwiftUI
+import Backpack_Common
+
+struct BPKDynamicStack<Content: View>: View {
+    @Environment(\.dynamicTypeSize) private var dynamicTypeSize
+
+    private let horizontalAlignment: HorizontalAlignment
+    private let verticalAlignment: VerticalAlignment
+    private let spacing: BPKSpacing
+    private let threshold: DynamicTypeSize
+    @ViewBuilder var content: () -> Content
+
+    public init(
+        horizontalAlignment: HorizontalAlignment = .center,
+        verticalAlignment: VerticalAlignment = .center,
+        spacing: BPKSpacing = .none,
```

Comment:
could it be the case that spacing needs to be different in vertical and horizontal enviroments? maybe you could create a small Struct that takes this as a required parameter like this:
```swift
struct HorizontalDynamicAlignment {
  let alignment: HorizontalAlignment
  let spacing: BPKSpacing
}

// and

struct VerticalDynamicAlignment {
  let alignment: VerticalAlignment
  let spacing: BPKSpacing
}
```
and then receive this instead:
```swift
public init(
    horizontalAlignment: HorizontalDynamicAlignment = .center,
    verticalAlignment: VerticalDynamicAlignment = .center,
    content: @escaping () -> Content
) {}
```
or even simpler than all this,
```swift
public init(
    horizontalAlignment: (HorizontalAlignment, BPKSpacing) = (.center, .none),
    verticalAlignment: (VerticalAlignment, BPKSPacing) = (.center, .none),
    content: @escaping () -> Content
) {}